Cargo.toml

```diff
@@ -239,7 +239,9 @@ mlua = { version = "0.6.2", default-features = false, features = ["lua54", "send
 mongodb = { version = "2.0.0-beta.3", default-features = false, features = ["tokio-runtime"], optional = true }
 async-nats = { version = "0.9.18", default-features = false, optional = true }
 no-proxy = { version  = "0.3.1", default-features = false, features = ["serialize"] }
-nom = { version = "6.1.2", default-features = false, optional = true }
+#https://github.com/wyyerd/pulsar-rs/issues/167
+nom = { version="=7.0.0-alpha1", default-features = false, features=["alloc"], optional = true}
+#nom = { version = "6.1.2", default-features = false, optional = true }
```

Comment:
I'm a little surprised this brought in a `nom` update. Did you mean to?

Reply:
Also is the `indexmap` change needed? I do know we have another PR that pins it too due to the circular dependency, but it doesn't _seem_ like this PR does?

Reply:
I can probably get rid of it by reverting the lock and not running cargo update.